activesupport/test/time_zone_test.rb

```diff
@@ -343,11 +343,19 @@ def test_iso8601_with_ambiguous_time
     assert_equal Time.utc(2014, 10, 25, 22, 0, 0), zone.parse("2014-10-26T01:00:00")
   end
 
-  def test_iso8601_with_invalid_value_parseable_by_date__iso8601
+  def test_iso8601_with_ordinal_date_value
+    zone = ActiveSupport::TimeZone["Eastern Time (US & Canada)"]
+
+    twz = zone.iso8601("21087")
+    assert_equal [0, 0, 0, 28, 03, 2021], twz.to_a[0, 6]
+    assert_equal zone, twz.time_zone
+  end
+
+  def test_iso8601_with_invalid_ordinal_date_value
     zone = ActiveSupport::TimeZone["Eastern Time (US & Canada)"]
 
     exception = assert_raises(ArgumentError) do
-      zone.iso8601("12936")
```

Comment:
Rather than remove this it’d be good to see a test for what now happens when this value is parsed.

Reply:
@ghiculescu No problem, `12936` is another invalid value (actually coming from a real-world case that led me to make [this PR](https://github.com/rails/rails/pull/41764) I can revert to using this value as the existing test will work, or test both this and `21367` as invalid ordinal values but that is sort of proving the same point twice. Perhaps it would have been better to have left the original invalid value in place?

Reply:
Nah that makes sense, thanks for the explanation.